src/amp-story-player-manager.js

```diff
@@ -99,8 +98,8 @@ export class AmpStoryPlayerManager {
     const doc = this.win_.document;
     const players = doc.getElementsByTagName('amp-story-player');
     for (let i = 0; i < players.length; i++) {
-      const playerEl = players[i];
-      const player = new AmpStoryPlayer(this.win_, playerEl);
+      const player = players[i];
+      player.init(this.win_);
```

Comment:
Good point, we can use `connectedCallback()` instead of calling `init()` manually. 

Re. `customElements.define('amp-story-player', AmpStoryPlayer);`, this happens in `amp-story-player.js` once the page is loaded